spec/serializer/defence_organisation_serializer_spec.rb

```diff
@@ -3,27 +3,32 @@
 RSpec.describe DefenceOrganisationSerializer do
   subject { described_class.new(defence_organisation).serializable_hash }
 
-  let(:defence_organisation) do
-    instance_double("DefenceOrganisation",
-                    id: "UUID",
-                    name: "The Johnson Partnership",
-                    address1: "104",
-                    address2: "Fleet Street",
-                    address3: "Westminster",
-                    address4: "London",
-                    address5: "GB",
-                    postcode: "EC4A 2AH")
+  context "with required fields" do
+    let(:defence_organisation_data) { JSON.parse(file_fixture("defence_organisation/required_fields.json").read) }
+    let(:defence_organisation) { DefenceOrganisation.new(body: defence_organisation_data) }
```

Comment:
My preference would be to have a fake DefenceOrganisation here so we are testing the serializer in isolation of the model. What are your thoughts?

Reply:
Yeah good point. And then we can use the fixtures to test the model itself.